.github/workflows/dis-cache-release.yml

```
@@ -0,0 +1,83 @@
+  build-release-flux-oci-latest:
+    name: Build latest from main
+    if: github.ref == 'refs/heads/main'
+    defaults:
+      run:
+        working-directory: ./services/dis-redis-operator/config
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
+      - name: Push latest flux oci image from main
+        uses: ./actions/flux/build-push-image
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/actions/flux/build-push-image": line 5: unexpected key "type" for definition of input "azure_tenant_id"

(action)